common/evmcompat/helpers.go

```diff
@@ -191,3 +191,85 @@ func SolidityPackedBytes(pairs []*Pair) ([]byte, error) {
 
 	return b.Bytes(), nil
 }
+
+func SolidityUnpackString(data string, types []string) ([]interface{}, error) {
```

Comment:
Add comment describing what this function does, and what the `types` are supported.